_pages/welcome-to-TTS/classes/intro-to-18f-infrastructure.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Intro to 18F Infrastructure
 navtitle: Infrastructure
+outdated: true
```

Comment:
I'm not sure this should be marked outdated. The referenced doc is pretty up to date, and it was all current. That said, it was current when Kimber and Erik were here. Perhaps the best path is to mark it outdated but also log an issue here or in the infra repo (or both) to revisit it when the infra-vival happens.

Reply:
Sure! I mostly marked it outdated because, technically speaking, there _is_ no 18F Infra team. There's a TTS infra team (and a page for it), but it seems confusing that there are two pages but not two teams.

Reply:
Behold! #1300

Reply:
Oh, I wasn't aware there was a separate TTS infra page. This looks pretty current, so I suspect effort has been going on in parallel. No worries, though, we can use #1300 to sort it out when Infra is back. LGTM!